automation/utils.ts

```diff
@@ -20,7 +20,8 @@ import * as _ from 'lodash';
 import * as path from 'path';
 import * as shellEscape from 'shell-escape';
 
-export const MSYS2_BASH = 'C:\\msys64\\usr\\bin\\bash.exe';
+export const MSYS2_BASH =
+	process.env.MSYSSHELLPATH || 'C:\\msys64\\usr\\bin\\bash.exe';
```

Comment:
For the record, this is not a new env var: it matches the existing var used here:
https://github.com/balena-io/balena-cli/blob/v12.30.3/patches/all/qqjs%2B%2Bexeca%2B0.10.0.dev.patch#L29